### ISSUE TITLE
[Bug]: metadata_path files MUST end in .yml

### ISSUE THREAD

**Issue by chrisjohnson00** [Kometa-Team/Kometa#1067]
### Version Number

1.17.3

### What branch are you on?

master

### Describe the Bug

I just upgraded from 1.15.1 to 1.17.3,
When I was running 1.15.1 my TV and Movie config files were named `tv.yaml` and `movies.yaml`, and life was wonderful.
When I updated my docker container to 1.17.3, without changing anything else, I got this error message:

```
|                             Loading Metadata File: config/movies.yaml                              |
|====================================================================================================|
| File Error: File does not exist /config/movies.yaml.yml                                            |
```
I changed my config and my file names to end in `.yml` and it started working correctly.

I would not expect a config that points to a file path to have the application make additions to the value, but honor what was provided as is.

**Comment by meisnate12** [comment]
this is fixed in 1.18.0

**Comment by chrisjohnson00** [comment]
Thanks!
